### PR TITLE
Pk1 d bugfix

### PR DESCRIPTION
**Pull request by nathalieGit**
A bug fixed: output fits file no longer opened if no forest selected to write in it

### CODE REVIEW

File: bin/do_Pk1D.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def compute_mean_delta(ll,delta,iv,zqso):
 #   Create root file
     if (args.out_format=='root') :
         from ROOT import TCanvas, TH1D, TFile, TTree, TProfile2D, TProfile
-        storeFile = TFile("Testpicca.root","RECREATE","PK 1D studies studies");
+        storeFile = TFile(args.out_dir+"Testpicca.root","RECREATE","PK 1D studies studies");
```
Comment (ngbusca) [inline bin/do_Pk1D.py]: It's safer to add a `/` in case the user doesn't include it:

    storeFile = TFile(args.out_dir+"/Testpicca.root"...

**Comment by ngbusca** [comment]
@nathalieGit could you take a look at the codacy recommendations?

**Comment by ngbusca** [comment]
@nathalieGit there's just a trailing space to be fixed and I can merge...

**Comment by nathalieGit** [comment]
done

**Pull request merged**